src/index.html

```diff
@@ -1,12 +1,351 @@
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item"><a class="nav__list-link is-active" href="#">Apple</a></li>
+          <li class="nav__list-item"><a class="nav__list-link" href="#">Samsung</a></li>
+          <li class="nav__list-item"><a class="nav__list-link" href="#">Smartphones</a></li>
+          <li class="nav__list-item">
+            <a
+              class="nav__list-link"
+              data-qa="nav-hover"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__list-item"><a class="nav__list-link" href="#">Gadgets</a></li>
+          <li class="nav__list-item"><a class="nav__list-link" href="#">Tablets</a></li>
+          <li class="nav__list-item"><a class="nav__list-link" href="#">Photo</a></li>
+          <li class="nav__list-item"><a class="nav__list-link" href="#">Video</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="card" data-qa="card">
+        <div class="card__info">
+          <img
+            class="card__info-img"
+            src="./images/imac.jpeg"
+            alt="imac"
+          >
+          <h3 class="card__info-text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+          <p class="card__info-code">Product code: 195434</p>
+        </div>
+
+        <div class="stars">
+          <div class="stars__container">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="stars__rate">
+            <p class="stars__rate-text">Reviews: 5</p>
+          </div>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-total">$2,199</p>
+        </div>
+
+        <a
+          class="card__btn"
+          href="#"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <div class="card__info">
+          <img
+            class="card__info-img"
+            src="./images/imac.jpeg"
+            alt="imac"
+          >
+          <h3 class="card__info-text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+          <p class="card__info-code">Product code: 195434</p>
+        </div>
+
+        <div class="stars">
+          <div class="stars__container">
```

Comment:
Where is the modifier classes?
```suggestion
          <div class="stars stars--4">
```